staging/src/k8s.io/client-go/dynamic/BUILD

```diff
@@ -8,34 +8,43 @@ load(
 
 go_test(
     name = "go_default_test",
-    srcs = ["client_test.go"],
+    srcs = [
+        "client_test.go",
+        "lister_test.go",
+    ],
     embed = [":go_default_library"],
     deps = [
         "//staging/src/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured:go_default_library",
+        "//staging/src/k8s.io/apimachinery/pkg/labels:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/runtime:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/runtime/schema:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/runtime/serializer/streaming:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/types:go_default_library",
+        "//staging/src/k8s.io/apimachinery/pkg/util/diff:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/watch:go_default_library",
         "//staging/src/k8s.io/client-go/rest:go_default_library",
         "//staging/src/k8s.io/client-go/rest/watch:go_default_library",
+        "//staging/src/k8s.io/client-go/tools/cache:go_default_library",
     ],
 )
 
 go_library(
     name = "go_default_library",
     srcs = [
         "interface.go",
+        "lister.go",
```

Comment:
can you place these in a subpackage?  `client-go/dynamic/dynamiclister`

Reply:
how about `client-go/dynamic/lister` ?

Reply:
> how about `client-go/dynamic/lister` ?

dynamiclister will autocomplete in IDEs without conflict.  Think http/httptest

Reply:
oh, nice! Since there is more than one `lister` in the system it makes sense.